web/pages/api/v0/comments.ts

```diff
@@ -51,7 +55,16 @@ export default async function handler(
     console.error(`Unknown error during validation: ${e}`)
     return res.status(500).json({ error: 'Unknown error during validation' })
   }
+
+  const { userId, limit } = params
+
   try {
+    if (userId) {
+      const comments = await getUserComments(userId, limit, 0)
+      res.setHeader('Cache-Control', 'max-age=15, public')
+      return res.status(200).json(comments)
+    }
+
     const contractId = await getContractId(params)
```

Comment:
Seems like if you specify both userId and contractId, it should get that user's comments from that contract

Reply:
Yep, missed that. Just pushed up a fix.